template/build/dev-server.js

```diff
@@ -1,3 +1,5 @@
+if (!process.env.NODE_ENV) process.env.NODE_ENV = 'development'
```

Comment:
@LinusBorg Just pinging on this. I think once we update this one bit, it should be good to merge. 🙂